bitnami/flux/README.md

```diff
@@ -41,6 +41,38 @@ The command deploys flux on the Kubernetes cluster in the default configuration.
 
 > **Tip**: List all releases using `helm list`
 
+## Configuration and installation details
+
+### Resource requests and limits
+
+Bitnami charts allow setting resource requests and limits for all containers inside the chart deployment. These are inside the `resources` value (check parameter table). Setting requests is essential for production workloads and these should be adapted to your specific use case.
+
+To make this process easier, the chart contains the `resourcesPreset` values, which automatically sets the `resources` section according to different presets. Check these presets in [the bitnami/common chart](https://github.com/bitnami/charts/blob/main/bitnami/common/templates/_resources.tpl#L15). However, in production workloads using `resourcePreset` is discouraged as it may not fully adapt to your specific needs. Find more information on container resource management in the [official Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/).
+
+### [Rolling VS Immutable tags](https://techdocs.broadcom.com/us/en/vmware-tanzu/application-catalog/tanzu-application-catalog/services/tac-doc/apps-tutorials-understand-rolling-tags-containers-index.html)
+
+It is strongly recommended to use immutable tags in a production environment. This ensures your deployment does not change automatically if the same tag is updated with a different image.
+
+Bitnami will release a new chart updating its containers if a new version of the main container, significant changes, or critical vulnerabilities exist.
+
+### Prometheus metrics
+
+This chart can be integrated with Prometheus by setting `*.metrics.enabled` (under the `helmController`, `imageAutomationController`, `imageReflectorController`, `kustomizeController`, `notificationController` and `sourceController` sections) to `true`. This will expose Flux native Prometheus ports in the containers. Additionally, it will deploy several `metrics` services, which can be configured under the `*.metrics.service` section (under the `agent`, `envoy`, `operator`, `hubble.peers` and `hubble.relay` sections). These `metrics` services will have the necessary annotations to be automatically scraped by Prometheus.
```

Comment:
```suggestion
This chart can be integrated with Prometheus by setting `*.metrics.enabled` (under the `helmController`, `imageAutomationController`, `imageReflectorController`, `kustomizeController`, `notificationController` and `sourceController` sections) to `true`. This will expose Flux native Prometheus ports in the containers. Additionally, it will deploy several `metrics` services, which can be configured under the `*.metrics.service` section (under the `helmController`, `imageAutomationController`, `imageReflectorController`, `kustomizeController`, `notificationController` and `sourceController` sections). These `metrics` services will have the necessary annotations to be automatically scraped by Prometheus.